openmdao/vectors/petsc_vector.py

```diff
@@ -107,9 +115,16 @@ def _initialize_data(self, root_vector):
         super(PETScVector, self)._initialize_data(root_vector)
 
         self._petsc = {}
+        self._imag_petsc = {}
```

Comment:
I made an initial attempt at this, and got failures, so something wasn't working right. Decided to put up a story for a later revisit.